src/Avalonia.Visuals/Media/TextFormatting/TextLayout.cs

```diff
@@ -73,6 +77,12 @@ public class TextLayout
         /// </summary>
         public double MaxHeight { get; }
 
+
+        /// <summary>
+        /// Gets the maximum number of text lines.
+        /// </summary>
+        public double MaxLines { get; }
```

Comment:
Why is this a double 😅?

Reply:
Hmm. Good catch. Didn't realize it is a double. Probably some copy paste error.

Reply:
Yes, it was a copy paste mistake, should I open a PR to fix that?

Reply:
Will fix it myself